### PR TITLE
Add content for railway template description

### PR DESCRIPTION
**Pull request by matifali**
Publishing content for https://railway.app/template/cUQ8_P

### CODE REVIEW

File: railway/railway.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  <a href="https://coder.com#gh-dark-mode-only">
+    <img src="https://raw.githubusercontent.com/coder/coder/main/docs/images/logo-white.png" style="width: 128px">
+  </a>
+  <h3>
+  Self-Hosted Remote Development Environments
+  </h3>
+  <a href="https://coder.com">
+    <img style="width: 650px" src="https://raw.githubusercontent.com/coder/coder/main/docs/images/banner-white.png">
+  </a>
+</div>
```
Comment (bpmct) [inline railway/railway.md:16]: Let's remove this. Looks too marketing-y

File: railway/railway.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+### Official
+
+- [**VS Code Extension**](https://marketplace.visualstudio.com/items?itemName=coder.coder-remote): Open any Coder workspace in VS Code with a single click
+- [**JetBrains Gateway Extension**](https://plugins.jetbrains.com/plugin/19620-coder): Open any Coder workspace in JetBrains Gateway with a single click
+
+### Community
+
+- [**Coder GitHub Action**](https://github.com/marketplace/actions/update-coder-template): A GitHub Action that updates Coder templates
+- [**Various Templates**](https://github.com/coder/coder/examples/templates/community-templates.md): Hetzner Cloud, Docker in Docker, and other templates the community has built.
```
Comment (bpmct) [inline railway/railway.md:81]: I'm 50/50 on whether we should keep this section. Can you make this call @matifali?

We could also remove all these headers (Documentation, community, etc) and just have bullet points

- [Coder on GitHub](#)
- [Coder Discord](#)
- [Coder docs](#)

File: railway/railway.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- Define development environments in Terraform
+  - EC2 VMs, Kubernetes Pods, Docker Containers, etc.
+- Automatically shutdown idle resources to save on costs
+- Onboard developers in seconds instead of days
+
+## Quickstart
+
+### 1. Fill the environment variables
+
+The current version of the template allows you to set an optional environment variable `CODER_WILDCARD_ACCESS_URL` which will allow you to forward ports from your workspace to a custom wildcard subdomain of the URL you provide. This is useful if you want to access a service running in your workspace from a browser. For example, if you set `CODER_WILDCARD_ACCESS_URL` to `*.coder.example.com`, you can access a service running on port 8080 in your workspace by visiting `8080--app-name-workspace-name-.coder.example.com` in your browser.
```
Comment (bpmct) [inline railway/railway.md:29]: Mention that you have to use a custom domain with Railway and cannot use railway's built-in domain for this wildcard.

File: railway/railway.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+&gt; Note: This is an optional step. If you do not set this variable, you will not be able to access services running in your workspace from a browser. For more information.
```
Comment (bpmct) [inline railway/railway.md:31]: Shouldn't `*gt;` be `>`?

**Review by bpmct — APPROVED** [review]

File: railway/railway.md (hunk 4 above)
Comment (matifali) [reply railway/railway.md:31]: It gets rendered as >. But I will verify

File: railway/railway.md (hunk 1 above)
Comment (matifali) [reply railway/railway.md:16]: ok. I thought we may do some marketing.

File: railway/railway.md (hunk 2 above)
Comment (matifali) [reply railway/railway.md:81]: I think can remove the Related section and keep the Official and Community.
Your suggestion is also good to just clean it up and give links.

File: railway/railway.md (hunk 3 above)
Comment (matifali) [reply railway/railway.md:29]: Linking do docs does not help here as on Railway only way to set the flags is through Env variables.

File: railway/railway.md (hunk 4 above)
Comment (matifali) [reply railway/railway.md:31]: my md formatter did that

**Pull request merged**